commands/init.go

```diff
@@ -28,10 +30,8 @@ func init() {
   > git remote add origin git@github.com:USER/REPO.git
 */
 func gitInit(command *Command, args *Args) {
-	if !args.IsParamsEmpty() {
```

Comment:
This check was never needed. `transformInitArgs` will bail out early as a no-op if the `-g` flag is not present, such as in the case where args are an empty list.